src/main/java/org/wso2/carbon/connector/amazonsqs/operations/message/ChangeMessageVisibility.java

```diff
@@ -0,0 +1,88 @@
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.carbon.connector.amazonsqs.operations.message;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.synapse.MessageContext;
+import org.wso2.carbon.connector.amazonsqs.connection.SqsConnection;
+import org.wso2.carbon.connector.amazonsqs.constants.Constants;
+import org.wso2.carbon.connector.amazonsqs.exception.SqsInvalidConfigurationException;
+import org.wso2.carbon.connector.amazonsqs.utils.Error;
+import org.wso2.carbon.connector.amazonsqs.utils.Utils;
+import org.wso2.carbon.connector.core.AbstractConnector;
+import org.wso2.carbon.connector.core.ConnectException;
+import org.wso2.carbon.connector.core.connection.ConnectionHandler;
+import org.wso2.carbon.connector.core.util.ConnectorUtils;
+import software.amazon.awssdk.awscore.AwsRequestOverrideConfiguration;
+import software.amazon.awssdk.core.RequestOverrideConfiguration;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityRequest;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityResponse;
+import software.amazon.awssdk.services.sqs.model.SqsException;
+
+import java.time.Duration;
+
+/**
+ * Implements change message visibility operation.
+ */
+public class ChangeMessageVisibility extends AbstractConnector {
+
+    @Override
+    public void connect(MessageContext messageContext) throws ConnectException {
+        try {
+            ConnectionHandler handler = ConnectionHandler.getConnectionHandler();
+            SqsConnection sqsConnection = (SqsConnection) handler
+                    .getConnection(Constants.CONNECTOR_NAME, Utils.getConnectionName(messageContext));
+            String queueUrl = Utils.createUrl(messageContext, sqsConnection);
+            String receiptHandle = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    Constants.RECEIPT_HANDLE_CONFIG);
+            String visibilityTimeout = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    Constants.VISIBILITY_TIME_OUT);
+            String apiCallTimeout = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    Constants.API_CALL_TIMEOUT);
+            String apiCallAttemptTimeout = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    Constants.API_CALL_ATTEMPT_TIMEOUT);
+            ChangeMessageVisibilityRequest.Builder changeMessageVisibilityRequestBuilder =
+                    ChangeMessageVisibilityRequest.builder().
+                    queueUrl(queueUrl).visibilityTimeout(Integer.valueOf(visibilityTimeout)).
+                    receiptHandle(receiptHandle);
+            if (StringUtils.isNotEmpty(apiCallTimeout) || StringUtils.isNotEmpty(apiCallAttemptTimeout)) {
```

Comment:
Shall we use `StringUtils.isNotBlank` to handle whitespaces as well? Please change the other occurrences as well.